Source/Particles/PulsarParameters.cpp

```diff
@@ -0,0 +1,91 @@
+#include <PulsarParameters.H>
+#include <AMReX_ParmParse.H>
+#include <AMReX_Print.H>
+#include <AMReX_RealVect.H>
+#include <AMReX_REAL.H>
+#include <AMReX_GpuQualifiers.H>
+
+namespace PulsarParm
+{
+   std::string pulsar_type;
+
+   AMREX_GPU_DEVICE_MANAGED amrex::Real omega_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real B_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real R_star;
+   AMREX_GPU_DEVICE_MANAGED amrex::Real dR_star;
+   AMREX_GPU_DEVICE_MANAGED int EB_external = 0;
+   AMREX_GPU_DEVICE_MANAGED int E_external_monopole = 0;
+   AMREX_GPU_DEVICE_MANAGED amrex::Vector<amrex::Real> center_star(3,0.0);
+   AMREX_GPU_DEVICE_MANAGED int verbose = 0;
+
+   void ReadParameters() {
+      amrex::ParmParse pp("pulsar");
+      pp.query("pulsarType",pulsar_type);
+      pp.query("omega_star",omega_star);
+      pp.queryarr("center_star",center_star);
```

Comment:
If `center_star` is a GpuArray, I don't think `queryarr` will work since it takes references to `std::vector` ...

I think we'll have to replace this line with:

```
amrex::Vector<amrex::Real> center_star_v(AMREX_SPACEDIM, 0.0);
pp.queryarr("center_star", center_star_v);
std::copy(center_star_v.begin(), center_star_v.end(), center_star.begin());
```

Reply:
done! Thank you